src/components/BaseInput/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import _ from 'underscore';
+import {
+    // eslint-disable-next-line no-restricted-imports
+    TextInput as RNTextInput,
+} from 'react-native';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    /** A ref to forward to the text input */
+    forwardedRef: PropTypes.func,
+};
+
+const defaultProps = {
+    forwardedRef: () => {},
+};
+
+const BaseInput = props => (
```

Comment:
Ah, sorry I hate to do this, but `BaseInput` is a pretty generic name and `BaseTextInput` is already taken... can we call this `RNTextInput.js` instead?

Reply:
ok done. thanks .

Reply:
Thanks for your patience here 🙇

Reply:
thanks.  😃